File: com.systemincloud.examples.tasks.javatask/src/test/java/tasks/constant/Constant2.java
package tasks.constant;

import com.systemincloud.modeler.tasks.javatask.api.JavaTask;
import com.systemincloud.modeler.tasks.javatask.api.OutputPort;
import com.systemincloud.modeler.tasks.javatask.api.annotations.JavaTaskInfo;
import com.systemincloud.modeler.tasks.javatask.api.annotations.OutputPortInfo;
import com.systemincloud.modeler.tasks.javatask.api.data.Int32;

@JavaTaskInfo(constant = true)
public class Constant2 extends JavaTask {

	@OutputPortInfo(name = "Out", dataType = Int32.class)
	public OutputPort out;

	@Override
	public void execute(int grp) {
		out.putData(new Int32(5));
	}

}

File: com.systemincloud.examples.tasks.pythontask/src/test/py/tasks/constant/Constant2.py
from sicpythontask.PythonTaskInfo import PythonTaskInfo
from sicpythontask.PythonTask import PythonTask
from sicpythontask.OutputPort import OutputPort
from sicpythontask.data.Int32 import Int32

@PythonTaskInfo(constant=True)
class Constant2(PythonTask):

    def __init_ports__(self):
        self.out = OutputPort(name="Out", data_type=Int32)

    def execute(self, grp):
        self.out.put_data(Int32(5))
